cmd/bnsd/x/account/init.go

```diff
@@ -57,6 +63,7 @@ func (*Initializer) FromGenesis(opts weave.Options, params weave.GenesisParams,
 		account := Account{
 			Metadata: &weave.Metadata{Schema: 1},
 			Domain:   d.Domain,
+			Owner:    d.Admin,
```

Comment:
Huh? Why is this unrelated change in here?

Reply:
Correct.
I have noticed this issue while testing manually. It's not related to no-op migration.

Reply:
okay, thanks. This looks like a bugfix of the `account` module that deserves a CHANGELOG entry

Reply:
This change does not belong here if backend was a team and not one person. Because I am alone right now, I allow myself for a big of controlled chaos :smiling_imp: 

I do not create changelog entries for all changes, only for those that are subjectively considered important. Do you think this change deserves a changelog entry?

Reply:
> Because I am alone right now, I allow myself for a big of controlled chaos 😈

Yeah, hard to get used to that but fine for me.

> are subjectively considered important. Do you think this change deserves a changelog entry?

The diff looks like creating a chain with `account` config in genesis is broken. This is very important for local development chains as soon as we start using the new functionality. A client dev needs to know: starting a new chain with bnsd 0.24.x and accounts config is broken, i.e. 0.25.0 must be used instead.